ttadmin/common/social.py

```diff
@@ -0,0 +1,28 @@
+from mastodon import Mastodon
+import tweepy
+
+from django.conf import settings
+
+mastodon = Mastodon(
+    client_id=settings.MASTO_CLIENT_ID,
+    client_secret=settings.MASTO_CLIENT_SECRET,
+    access_token=settings.MASTO_ACCESS_TOKEN,
+    api_base_url=settings.MASTO_BASE_URL
+)
+
+tw_auth = tweepy.OAuthHandler(settings.TWITTER_CONSUMER_KEY, settings.TWITTER_CONSUMER_SECRET)
+tw_auth.set_acces_token(settings.TWITTER_TOKEN, settings.TWITTER_TOKEN_SECRET)
+twitter = tweepy.API(tw_auth)
+
+def post_to_social(qs):
```

Comment:
looking good! thank you!!!

couple thoughts:

- it'd be nice to have two separate functions: `post_to_mastodon` and `post_to_twitter` that `post_to_social` can then use
- what if the list of new users exceeds the limit for twitter?
- i guess the same question for mastodon but it seems very very unlikely

Reply:
I guess we should post into two different posts, the question is, should the welcome part be identical?

Also, should `post_to_social` just pass the queryset on to the other functions?